api/src/main/java/com/fit_goal/domain/UserNotification.java

```diff
@@ -0,0 +1,17 @@
+package com.fit_goal.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class UserNotification {
```

Comment:
Instead of the NoArgsConstructor it might be more useful to have AlllArgsConstructor

Reply:
refactored